flowbench/BenchAsyncFileWriteCheckerLRU.cpp

```diff
@@ -0,0 +1,48 @@
+#include "benchmark/benchmark.h"
+#include "crc32/crc32c.h"
+#include "flow/Hash3.h"
+#include "flow/xxhash.h"
+#include "flowbench/GlobalData.h"
+#include "fdbrpc/AsyncFileWriteChecker.actor.h"
+
+#include <stdint.h>
+
+static void lru_test(benchmark::State& state) {
+	int run = 10000;
+	std::set<uint32_t> exist;
+	int limit = 150000000; // 600GB file
+	AsyncFileWriteChecker::LRU lru("TestLRU");
+
+	// Benchmark
+	for (auto _ : state) {
+		for (int i = 0; i < run; ++i) {
+			double r = deterministicRandom()->random01();
+			// [0. 0,45] remove
+			// [0.45, 0.5] truncate
+			// [0.5, 1] update
+			if (exist.size() < 2 || r > 0.5) {
+				// to add/update
+				uint32_t page = deterministicRandom()->randomInt(0, limit);
+				// change the content each time
+				auto wi = AsyncFileWriteChecker::WriteInfo();
+				wi.checksum = deterministicRandom()->randomInt(0, INT_MAX);
```

Comment:
is checksum going to be used in the benchmark? if not, we don't need to spend time calling random function, thus more accurate in measuring LRU time overhead.